data/transactions/logic/mocktracer/tracer_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package mocktracer
+
+import (
+	"testing"
+
+	"github.com/algorand/go-algorand/data/transactions/logic"
+	"github.com/algorand/go-algorand/test/partitiontest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMockTracerIsEvalTracer(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	var tracer logic.EvalTracer = &Tracer{}
+	require.NotNil(t, tracer)
+}
```

Comment:
I think this literally can't fail.  It's intended to be a test of the mock?  I'd just skip it.

Reply:
This was helpful to make sure the mock tracer still implements logic.EvalTracer as I've been modifying its functions. It was useful locally but I'll remove it from the PR